Classes/UT3HellfireSPMAShellTrail.uc

```diff
@@ -39,7 +32,7 @@ defaultproperties
 		ParticlesPerSecond=0.0
 		InitialParticlesPerSecond=0.0
 		DrawStyle=PTDS_AlphaBlend
-		Texture=Texture'SPMASmoke'
+        Texture=Texture'UT3SPMATex.Smoke.SPMASmoke'

```

Comment:
Thanks for tabs to spaces, but if you fix it here, please fix it in the surrounding lines as well!